dScripts/Darkitect.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+#include "CppScripts.h"
+
+class Darkitect : public CppScripts::Script
```

Comment:
This doesnt need to inherit the CppScripts::Script interface since it is a helper class.  This can safely be removed along with the include

Reply:
ah, yes! I made note of that and lost access to my server yesterday before I could make that change